lnwallet/btcwallet/btcwallet.go

```diff
@@ -1554,7 +1554,8 @@ func unminedTransactionsToDetail(
 //
 // This is a part of the WalletController interface.
 func (b *BtcWallet) ListTransactionDetails(startHeight, endHeight int32,
-	accountFilter string) ([]*lnwallet.TransactionDetail, error) {
+	accountFilter string, indexOffset uint,
```

Comment:
nit: we should always used sized integers. So should be `uint32, int32` respectively.